widget/migrations/0004_auto_20151125_1011.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.core.validators
+
+
+class Migration(migrations.Migration):
```

Comment:
@danaspiegel This is here by mistake, because when I wrote the test cases I created its branch that doesn't have the widget_host_url field changed to CharField instead of URLField. So I changed it for this pull request and seems that I reverted the change in models.py but didn't revert this change. So This file shouldn't be merged.